build.gradle

```diff
@@ -23,8 +23,6 @@ allprojects {
         mavenCentral()
         maven { url "https://jitpack.io" }
         maven { url "https://oss.sonatype.org/content/repositories/snapshots/" }
-        maven { url "https://kotlin.bintray.com/kotlinx" }
-        maven { url "https://dl.bintray.com/jetbrains/kotlin-native-dependencies" }
```

Comment:
Do not exist anymore because of the bintray shutdown

Reply:
👍  Thanks
Overall this repository needs a lot of clean up, most of the things were hacked together and sample was copied from RxRedux but evolved overtime so there are still some unused dependencies.

So thanks for taking the first step 👍